client/trino-client/src/main/java/io/trino/client/auth/kerberos/ContextBasedSubjectProvider.java

```diff
@@ -46,7 +46,7 @@ public void refresh()
         if (credentials.size() > 1) {
             throw new ClientException("Invalid Credentials. Multiple Kerberos Credentials found.");
         }
-        KerberosTicket kerberosTicket = getOnlyElement(credentials);
```

Comment:
IMO this was the right way to write this. 
I'd switch over only when `Iterables.getOnlyElement` is really deprecated, or rather  -- would consider introducing our method to do the same.

Note that we already did the check `if (credentials.size() > 1)`. If we don't use `getOnlyElement` short version, people will gravitate towards the second short thing: `.get(0)` which is less future-proof.